internal/services/appservice/helpers/enums.go

```diff
@@ -0,0 +1,6 @@
+package helpers
+
+const (
+	PublicNetworkAccessEnabled  string = "Enabled"
+	PublicNetworkAccessDisabled string = "Disabled"
+)
```

Comment:
Is there a Swagger issue tracking these being omitted?

Reply:
Comment added to the tracking issue 👍